public_desktop/desktop-app.js

```diff
@@ -92,6 +94,42 @@ function startDesktopApp() {
 
 	debug = gGebug( 'desktop:browser' );
 
+	try {
+		var buildEditorContextMenu = electron.remote.require( '../desktop/lib/menu/editor-context-menu' );
+		var buildGeneralContextMenu = electron.remote.require( '../desktop/lib/menu/general-context-menu' );
```

Comment:
Might be nice to avoid the traversal and just be able to require `lib/menu/editor-context-menu`. We may need to set `NODE_PATH` for this to work?